gems/aws-sigv4/lib/aws-sigv4/signer.rb

```diff
@@ -202,7 +202,7 @@ def initialize(options = {})
       #
```

Comment:
would need to add a changelog for sigv4 gem as well

Reply:
The signer changes do not have any new functionality or fixes. The slight refactor is just included. Next version bump of that gem will include that refactor.